src/index.html

```diff
@@ -4,14 +4,190 @@
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no,
+      initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <div class="group__input-block">
+          <label>
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label>
+            Name:
+            <input
+              type="text"
+              name="name"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              name="age"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label>
+            Full date of birth:
+            <input
+              type="date"
+              name="dateOfBirth"
+            >
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="termConditions"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>Registration</legend>
+        <div class="group__input-block">
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+        <div class="group__input-block">
+          <label>
+            Password:
+            <input
+              type="password"
+              name="password"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+        <div class="group__input-block">
+          <label>
+            Do you love cats?
+            <input
+              type="radio"
+              value="yes"
+              name="loveCats"
+            > Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              value="no"
+              name="loveCats"
+            > No
+          </label>
```

Comment:
Let's not put text in one line with self-closing text
```suggestion
            <input
              type="radio"
              value="yes"
              name="loveCats"
            > 
            Yes
          </label>
          <label>
            <input
              type="radio"
              value="no"
              name="loveCats"
            > 
            No
          </label>
```